plugins/inventory/now.py

```diff
@@ -364,7 +369,7 @@ def construct_sysparm_query(query, is_encoded_query):
 def fetch_records(table_client, table, query, fields=None, is_encoded_query=False):
     snow_query = dict(
         # Make references and choice fields human-readable
-        sysparm_display_value=True,
```

Comment:
Please use the value of `show_values` to set this parameter. You risk breaking the inventories of other users.

Reply:
By default it needs to be `True`. Change it to `all` only if `show_values` is `true`